packages/tooling/fast-tooling/src/css-data.ts

```diff
@@ -0,0 +1,18730 @@
+/**
+ * This file is generated from build/generate-mdn-data-files.js
+ * any modifications will be overwritten.
+ */
```

Comment:
Do we want this built file checked in?

Reply:
We do, however I've changed this around to be a js file so that I can import it and provide a check to see whether it should be updated. This provides a level of stability should MDN change their formatting which would break our current utilities expectations.